amundsen_application/static/css/_variables-default.scss

```diff
@@ -130,3 +129,39 @@ $spacer-1: $spacer-size;
 $spacer-2: $spacer-size * 2;
 $spacer-3: $spacer-size * 3;
 $spacer-4: $spacer-size * 4;
+
+// Elevations (from LPL)
```

Comment:
nit: can remove LPL references since it doesn't mean anything for the community, but not blocking at all.

Reply:
Good Catch!

I will do it in the next PR, thanks!